src/components/Button.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const Button = (props) => {
 
     return (
-        <button>{props.name}</button>
+        <button type="button" onClick={props.onClick}>{props.name}</button>
```

Comment:
make the type a property of your `Button` component so that it can be configured if needed